src/Microsoft.AspNet.OData.Shared/EdmPatchMethodHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.OData.Edm;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.OData
+{    
+   
+    /// <summary>
+    /// Handler Class to handle users methods for create, delete and update
+    /// </summary>
+    public abstract class EdmPatchMethodHandler
```

Comment:
>EdmPatchMethodHandler [](http://example.com/codeflow?start=26&length=21)

Since this is not just used for Patch, should we consider renaming to something like "UpdateHandler" or "DataModificationHandler"?